evm/test/EndpointStructs.t.sol

```diff
@@ -29,6 +29,7 @@ contract TestEndpointStructs is Test {
         bytes4 wh_prefix = 0x9945FF10;
         EndpointStructs.EndpointMessage memory em = EndpointStructs.EndpointMessage({
             sourceManagerAddress: hex"042942FAFABE",
+            recipientManagerAddress: hex"042942FAFABE",
```

Comment:
Nit - can we use a unique address for recipientManagerAddress?

Reply:
sure